changelog/unreleased/changes/2137--more-intuitive-makefile.md

```diff
@@ -0,0 +1 @@
+Enhance the Makefile for the cloud deployment to have more streamlined commands.
```

Comment:
I don't think this needs a changelog entry—after all, the whole cloud deployment stuff was not yet released to begin with.

Reply:
Makes sense! I deleted it, you can re-enable the auto-merge 😉